crates/solidity/outputs/npm/package/src/generated/index.js

```diff
@@ -241,7 +241,6 @@ if (!nativeBinding) {
   throw new Error(`Failed to load native binding`);
 }
 
-const { legacy, syntax$nodes } = nativeBinding;
+const { legacy } = nativeBinding;
```

Comment:
this PR moves everything back to `legacy` namespace. Not sure if that is intended? I suggest moving them to match (roughly) the rust namespaces. For example, TypeScript's `syntax$nodes` matched Rust's `syntax::nodes`. This makes it easier to reason about code, and prevents future collisions.